app/controllers/state_file/questions/verification_code_controller.rb

```diff
@@ -67,7 +67,6 @@ def redirect_into_login(intake, existing_intake)
         )
         @form.intake = existing_intake
         intake.destroy unless intake.id == existing_intake.id
-        sign_in existing_intake
```

Comment:
This was wrong! We should not sign users in until we have verified their SSN, or determined that they don't need to verify their ssn because they have not yet transferred data from directfile.